src/editor.rs

```diff
@@ -9,7 +9,7 @@ use std::str;
 use tempfile::NamedTempFile;
 use tracing::warn;
 
-use crate::VetError;
+use crate::errors::EditError;
```

Comment:
We might want to eventually change the `io::ERror` returning methods to return `EditError`, but this is fine for now.